content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -382,6 +382,81 @@ You can also see the `retainKeys` strategy in the [OpenApi spec](https://raw.git
 And you can see the `retainKeys` strategy in the
 [Kubernetes API documentation](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deploymentspec-v1-apps).
 
+## Use subresource flag with kubectl patch command to update the deployment
+
+In kubernetes v1.24, kubectl commands like get, patch, edit and replace will now contain a new flag `--subresource=[subresource-name]` which will allow fetching and updating `status` and `scale` subresources for all resources (built-in and CRs) that support these subresources.
+
+Here's the configuration file for a Deployment that has two replicas.
+
+{{< codenew file="application/deployment-subresource.yaml" >}}
+
+Create the Deployment:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/deployment-subresource.yaml
+```
+
+View the Pods associated with your Deployment:
+
+```shell
+kubectl get pods
+```
+
+In the output, you can see that Deployment has two Pods.
+
+```
+NAME                                READY   STATUS    RESTARTS   AGE
+subresource-demo-777d58f9d8-7wzff   1/1     Running   0          28s
+subresource-demo-777d58f9d8-fkf4l   1/1     Running   0          28s
+```
+
+Now, Patch your Deployment with `--subresource=[subresource-name]` flag:
+
+```shell
+kubectl patch deployment subresource-demo --subresource='scale' --type='merge' -p '{"spec":{"replicas":3}}'
+```
+
+you can see following output, which means your deployment is successfully patched:
+
+```shell
+scale.autoscaling/subresource-demo patched
+```
+
+View the Pods associated with your patched Deployment:
+
+```shell
+kubectl get pods
+```
+
+In the output, you can see one new pod is created, so now you have 3 running pods.
+
+```
+NAME                                READY   STATUS    RESTARTS   AGE
+subresource-demo-777d58f9d8-7wzff   1/1     Running   0          2m10s
+subresource-demo-777d58f9d8-fkf4l   1/1     Running   0          2m10s
+subresource-demo-777d58f9d8-v8cjk   1/1     Running   0          19s
+```
+
+View the patched Deployment:
+
+```shell
+kubectl get deployment subresource-demo -o yaml
+```
+
+```yaml
+spec:
```

Comment:
```suggestion
...
spec:
```
You could also use jsonpath output to select just the status.

Reply:
we can use jsonpath but on using it give output in the below format:

```
$ kubectl get deployment subresource-demo  -o=jsonpath='{.status}'
{"availableReplicas":3,"conditions":[{"lastTransitionTime":"2022-08-17T07:35:17Z","lastUpdateTime":"2022-08-17T07:35:57Z","message":"ReplicaSet \"subresource-demo-777d58f9d8\" has successfully progressed.","reason":"NewReplicaSetAvailable","status":"True","type":"Progressing"},{"lastTransitionTime":"2022-08-18T08:52:25Z","lastUpdateTime":"2022-08-18T08:52:25Z","message":"Deployment has minimum availability.","reason":"MinimumReplicasAvailable","status":"True","type":"Available"}],"observedGeneration":2,"readyReplicas":3,"replicas":3,"updatedReplicas":3}
```

IMO it seems good and also  YAML content is used to show the output in the corresponding docs, so for the uniformity of the docs, we can write in this way.